js/activity.js

```diff
@@ -186,6 +186,7 @@ const doAnalyzeProject = function() {
 };
 
 class Activity {
+    
```

Comment:
please don't add unrelated changes

Reply:
yes, fixed it. Thank you for the review.